lib/membrane/srtp/encryptor.ex

```diff
@@ -63,7 +64,9 @@ defmodule Membrane.SRTP.Encryptor do
     }
 
     :ok = ExLibSRTP.add_stream(state.srtp, policy)
-    {{:ok, redemand: :output}, Map.put(state, :policies, [policy])}
+
+    {{:ok, buffer: {:output, Enum.reverse(state.queue)}},
```

Comment:
What is the purpose of the queue?

If there are no policies we would end up caching the buffers and when we get the handshake we just send them all without encryption?

Reply:
The problem is that SRTP policy may come via output pad and till now we could hold demands until we receive it, which is not possible with auto demands, so we have to buffer them and encrypt them once we get the policy.

> we just send them all without encryption

good point, fixed :P